python/ray/air/_internal/accelerator_utils/nvidia_gpu.py

```diff
@@ -0,0 +1,78 @@
+import os
+from typing import List
+
+import torch
+
+import ray
+from ray.air._internal.accelerator_utils.device_manager import TorchDeviceManager
+
+
+class CUDATorchDeviceManager(TorchDeviceManager):
```

Comment:
Yes, we should instantiated a device manager for each work. 

Since the `DeviceManager` is currently supported only for PyTorch, a DeviceManager instance is instantiated for each TrainWorker during the `_TorchBackend.start_training()` process,  and stored it in the `_TrainSession`.